devnotes/payload_starting.md

```diff
@@ -191,6 +194,31 @@ pflash -P PAYLOAD -r /tmp/skiboot.bin
 tail -c +$((0x1001)) skiboot.bin | unxz > skiboot.lid
 ```
 
+### Building skiboot.lid
+
+   1. Clone the skiboot repository
+   ```
+   git clone https://git.raptorcs.com/git/talos-skiboot
+   ```
+   
+   2. Checkout correct revision
+   ```
+   git checkout 9858186353f2203fe477f316964e03609d12fd1d
```

Comment:
@IgorBagnucki it may be obvious but `cd talos-skiboot` is missing. It could be added here in this block for completness.

Reply:
Other than that, the build was successful for me. So let's just fix this and it's ready to merge.